src/components/SpiceList.js

```diff
@@ -1,22 +1,31 @@
-import React from 'react'
+import React from 'react';
+import SpiceItem from './SpiceItem';
 // TODO: import the SpiceItem component here
 
 // class component
 class SpiceList extends React.Component {
-
   renderSpices() {
     // TODO: update this function to return an array of SpiceItem components
     // If you get an error message about a 'key' prop, make sure to fix it before submitting your code!
+    const spices = this.props.spices;
+    return spices.map((spice) => {
```

Comment:
Here, there's not really a need to create a new variable for spices, you can just as easily call map on `this.props.spices` directly: 

```
return this.props.spices.map(spice => ...
```

If you prefer writing it this way, that's fine - just wanted to point that out 😄